### ISSUE TITLE
Follow icon visibility 

### ISSUE THREAD

**Issue by Fastidious** [nolanlawson/pinafore#2287]
When using pitch dark theme (perhaps the same for others colour themes), the follow/unfollow icon is hard to see.

**Comment by Fastidious** [comment]
I should have done this when I opened the issue. This is how it looks under Safari on iOS:

![FD7E42D9-5B7D-48EC-8DF2-5D0F8E8A0910](https://user-images.githubusercontent.com/8352292/205772095-e1e59e72-cafc-4973-b942-5d91daa83b58.jpeg)

If #2277 corrects that, hurrah! 😊

**Comment by NickColley** [comment]
Here's how it looks in #2277 

![image](https://user-images.githubusercontent.com/2445413/205780212-d6f31b73-e8af-4701-99bb-1923b4fe1468.png)

**Comment by NickColley** [comment]
Fixed in #2324 so keep your eyes peeled for the release.

**Comment by Fastidious** [comment]
@NickColley confirmed. It looks awesome, thanks!
